package-lock.json

```diff
@@ -22,6 +21,12 @@
         "web-vitals": "^2.1.4"
       },
       "devDependencies": {
+        "@testing-library/dom": "^9.0.1",
+        "@testing-library/react": "^14.0.0",
+        "@types/react-test-renderer": "^18.0.0",
+        "cypress": "^12.7.0",
+        "jest": "^27.5.1",
```

Comment:
please add package-lock into gitignore

Reply:
package-lock added to gitignore, thank you for you comment.